sysutils/ndcctools/Portfile

```
@@ -13,7 +13,7 @@ categories              sysutils
 platforms               darwin
 license                 MIT
 
-maintainers             nd.edu:pivie \
+maintainers             nd.edu:dthain \
```

Please also add your Github handle.  Most changes to ports these days are happening on Github, and with your Github handle here, you will get pinged on any pull requests opened against ports where you are the maintainer.

```suggestion
maintainers             {nd.edu:dthain @dthain} \
```